test/suite/test_timestamp26.py

```diff
@@ -576,6 +559,10 @@ def test_in_memory_ts(self):
                 self.session.commit_transaction('commit_timestamp=' + self.timestamp_str(1))
 
         # Commit without a timestamp.
```

Comment:
Stale comment? Or does it below belong the new text?

Reply:
Fixed in cc7ecf0.